### ISSUE TITLE
H5178 index out of range

### ISSUE THREAD

**Issue by PierreAronnax** [Bluetooth-Devices/govee-ble#48]
**Describe the bug**
I see you have merged https://github.com/Bluetooth-Devices/govee-ble/pull/47 without the suggested change in my review.
For H5178 it must be
`temp, humi = decode_temp_humid(data[3:6])`
just like the old line was
`packet_5178 = data[3:6].hex()`

**Additional context**
```
2022-12-15 10:11:17.492 ERROR (MainThread) [homeassistant.components.govee_ble] Unexpected error updating B51782BC8 data: index out of range
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/bluetooth/passive_update_processor.py", line 124, in _async_handle_bluetooth_event
    update = self._update_method(service_info)
  File "/usr/local/lib/python3.10/site-packages/bluetooth_sensor_state_data/__init__.py", line 66, in update
    self._start_update(data)
  File "/usr/local/lib/python3.10/site-packages/govee_ble/parser.py", line 78, in _start_update
    self._process_mfr_data(address, local_name, mfr_id, mfr_data, service_uuids)
  File "/usr/local/lib/python3.10/site-packages/govee_ble/parser.py", line 172, in _process_mfr_data
    temp, humi = decode_temp_humid(data[3:5])
  File "/usr/local/lib/python3.10/site-packages/govee_ble/parser.py", line 37, in decode_temp_humid
    (temp_humid_bytes[0] << 16) + (temp_humid_bytes[1] << 8) + temp_humid_bytes[2]
IndexError: index out of range
```

**Comment by bdraco** [comment]
I don't see your suggested change in the review

**Comment by bdraco** [comment]
I reverted #47

**Comment by PierreAronnax** [comment]
No?
![afbeelding](https://user-images.githubusercontent.com/1077773/207937237-47c02cc3-5f89-4c3b-ba29-01c33d65cf1d.png)

**Comment by bdraco** [comment]
Your comment is still pending

**Comment by PierreAronnax** [comment]
My bad, first time using that feature... (not really intuitive)

**Comment by bdraco** [comment]
thanks fixed in #50
